### PR TITLE
add WMArchive related views and field

### PR DESCRIPTION
**Pull request by ticoann**
add the view for checking upload is done by given workflow.
add jobtype and jobstatus for fwjr

### CODE REVIEW

**Comment by ticoann** [comment]
@vkuznet, @amaltaro, Please review

File: src/python/WMCore/Services/WMArchiver/DataMap.py [hunk 1]
```diff
@@ -247,6 +252,8 @@ def createArchiverDoc(job_id, fwjr, version=None):
     newfwjr['meta_data'] = {'agent_ver': version,
                          'host': socket.gethostname().lower(),
                          'fwjr_id': job_id,
-                         'ts': int(time.time())
+                         'jobtype': jobtype,
```
Comment (vkuznet) [inline src/python/WMCore/Services/WMArchiver/DataMap.py]: @ticoann , Seangchan does it means we need to tweak WMArchive schema again and add jobtype and jobstate to it, see https://github.com/dmwm/WMArchive/blob/master/test/data/samples/fwjr_production.py#L4

Comment (ticoann) [inline src/python/WMCore/Services/WMArchiver/DataMap.py]: Yes, I will update new schema after running the test. With this patch I can extract all different kinds of fwjr by type and its status. I will change necessary schema changes and create the new pull request in WMArchive.

**Comment by ticoann** [comment]
tested now in vm, merging it

**Pull request merged**